wolfcrypt/src/aes.c

```diff
@@ -3367,11 +3405,6 @@ int wc_AesSetIV(Aes* aes, const byte* iv)
 
 #endif
 
-enum {
-    NONCE_SZ = 12,
```

Comment:
John, there are two places in the code that still use `NONCE_SZ`. Both are async cases. If you run `./async-check.sh` those errors show up:

```
wolfcrypt/src/aes.c:8528:70: error: use of undeclared identifier 'NONCE_SZ'
                            sz >= WC_ASYNC_THRESH_AES_GCM && ivSz == NONCE_SZ) {
                                                                     ^
wolfcrypt/src/aes.c:8881:70: error: use of undeclared identifier 'NONCE_SZ'
                            sz >= WC_ASYNC_THRESH_AES_GCM && ivSz == NONCE_SZ) {
                                                                     ^
2 errors generated.
```